api/api.spec.yaml

```diff
@@ -153,6 +153,57 @@ paths:
         enum: [0, 1, 2]
         description: Parameter to control verbosity of returned data object
         required: false
+  /payloads/{request_id}/archiveLink:
+    get:
+      description: Get the download URL for a payload's archive
+      parameters:
+        - name: request_id
+          in: path
+          description: A unique value identifying this payload.
+          required: true
+          type: string
```

Comment:
Should this be it's own endpoint rather than part of `/{request_id}/`? If not, maybe we can take out that `request_id` parameter since it's already part of the url? :thinking:

Reply:
Also, if we decide to keep the `request_id` parameter, might be a good idea to have the `request_id` format as an UUID.
 
s/o link: [here](https://stackoverflow.com/a/50210623)

Reply:
I think you're right. I removed the `request_id` parameter here. I think it makes sense if `/payloads/{request_id}` returns the info for that payload, then `/payloads/{request_id}/archiveLink` would return the archive link associated with that payload.